bin/tldr

```diff
@@ -4,7 +4,7 @@ const program = require('commander');
 const pkg = require('../package');
 const Tldr = require('../lib/tldr');
 const config = require('../lib/config');
-const platform = require('../lib/platform');
+const Platform = require('../lib/platform');
```

Comment:
I think the discussion in #337 got this wrong, where naming this `Platform` is against standard conventions, where if it's capitalized, then it's assumed that we're referring to a class object, vs a regular object. I'd personally say rename the file to `lib/platforms` and have the import be `const platforms = require('../lib/platforms')`.